.jenkins/actions/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+envloc=$1
+RUN_CMD_FILE=$2
+
+. ${envloc}/env/machineEnvironment.sh
```

Comment:
You use `.` here and `source` in a later script.

Reply:
Not sure what you are referring to, but this script is now removed